heron/executor/src/python/heron_executor.py

```diff
@@ -325,6 +337,47 @@ def _get_metricsmgr_cmd(self, metricsManagerId, sink_config_file, port):
 
     return metricsmgr_cmd
 
+  def _get_metrics_cache_cmd(self):
+    ''' get the command to start the metrics manager processes '''
+    metricscachemgr_main_class = 'com.twitter.heron.metricscachemgr.MetricsCacheManager'
+
+    metricscachemgr_cmd = [os.path.join(self.heron_java_home, 'bin/java'),
+                           # We could not rely on the default -Xmx setting, which could be very big,
+                           # for instance, the default -Xmx in Twitter mesos machine is around 18GB
+                           '-Xmx1024M',
```

Comment:
For the calculation: Make sure eventually the overall ram required for a container will not exceed the ram requested, especially by adding this one more process.